RESTserver/src/main/java/com/softserve/if072/restservice/controller/ProductController.java

```diff
@@ -60,11 +60,16 @@ public ProductController(ProductService productService) {
     @GetMapping(value = "/user/{userId}")
     @ResponseBody
     @ResponseStatus(value = HttpStatus.OK)
-    public List<Product> getAllProductsByUserId(@PathVariable int userId) throws DataNotFoundException {
+    public List<Product> getAllProductsByUserId(@PathVariable int userId) {
 
-        List<Product> products = productService.getAllProducts(userId);
-        LOGGER.info("All products were found");
-        return products;
+        try {
+            List<Product> products = productService.getAllProducts(userId);
+            LOGGER.info("All products were found");
+            return products;
+        } catch (DataNotFoundException e) {
+            LOGGER.error(e.getMessage());
+            return new ArrayList<Product>();
```

Comment:
It does not make sense, because service can return empty List. Throw exception is redundant operation